app/main/views/briefs.py

```diff
@@ -0,0 +1,41 @@
+from flask import render_template, request, redirect, flash
+from flask_login import current_user
+
+from ..helpers import login_required
+from ..helpers.briefs import (
+    get_brief,
+    check_supplier_is_eligible_for_brief,
+    send_brief_clarification_question,
+)
+from ...main import main
+from ... import data_api_client
+
+
+@main.route('/opportunities/<int:brief_id>/ask-a-question', methods=['GET', 'POST'])
+@login_required
+def ask_brief_clarification_question(brief_id):
+    brief = get_brief(data_api_client, brief_id)
+    check_supplier_is_eligible_for_brief(brief, current_user.supplier_id)
+
+    error_message = None
+    clarification_question_value = None
+
+    if request.method == 'POST':
+        clarification_question = request.form.get('clarification-question', '').strip()
+        if not clarification_question:
+            error_message = "Question cannot be empty"
+        elif len(clarification_question) > 5000:
```

Comment:
On the API side there is a 100 word limit and no character limit. We should decide what the limits should be and use the same.